src/pages/home/report/ReportActionItem.js

```diff
@@ -47,6 +59,16 @@ class ReportActionItem extends Component {
             || !_.isEqual(this.props.action, nextProps.action);
     }
 
+    /**
+     * Is the current user in the ReportActionContextMenu beta?
+     *
+     * @returns {Boolean}
+     */
+    isInReportActionContextMenuBeta() {
```

Comment:
Sorry, are you talking about `permissions.js` or something else? Could you maybe point me to an example of what you mean in web?

Reply:
Yes, similar to permissions.js or [beta.jsx](https://github.com/Expensify/Web-Expensify/blob/399c0ae6560201f10824f7ac266f64eee9f1cc74/concierge/js/lib/Beta.jsx#L1) 

If we are eventually removing this and this is the only beta though, I do not think this is needed.

Reply:
Yeah, we'll eventually remove this. We could do that, but since this is the only beta for now and we'll remove it soon, maybe we just stick with the simplest solution to avoid over-engineering anything :)

Reply:
Awesome, I agree! No need to overengineer this for a single beta!